core/dns.go

```diff
@@ -79,49 +78,6 @@ func (dnsResolver *DNSResolverImpl) LookupTXT(hostname string) ([]string, time.D
 	return txt, rtt, err
 }
 
-// LookupHost sends a DNS query to find all A/AAAA records associated with
```

Comment:
So I forget which PR removed this, then #505 added it back in, then we reverted #505, so again it's not in master. Once #505 gets re-applied these metrics can be recorded again ._.